### PR TITLE
[Trainer] Change num_train_epochs default value

### PR DESCRIPTION
**Pull request by DesmonDay**
### PR types
Others

### PR changes
Others

### Description
Change num_train_epochs default value.

### CODE REVIEW

**Review by lugimzzz — DISMISSED** [review]
lgtm

**Review by wawltor — DISMISSED** [review]
LGTM

File: paddlenlp/trainer/training_args.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ class TrainingArguments:
             The epsilon hyperparameter for the [`AdamW`] optimizer.
         max_grad_norm (`float`, *optional*, defaults to 1.0):
             Maximum gradient norm (for gradient clipping).
-        num_train_epochs(`float`, *optional*, defaults to 3.0):
```
Comment (ZHUI) [inline paddlenlp/trainer/training_args.py:130]: 中文文档同步修改

**Review by ZHUI — APPROVED** [review]
LGTM

**Pull request merged**